.github/workflows/reusable-performance.yml

```diff
@@ -298,6 +297,11 @@ jobs:
       - name: Compare results
         run: node ./tests/performance/compare-results.js ${{ runner.temp }}/summary.md
 
+      - name: Check date time
+        run: echo "COMMITTED_AT_TEST=$(git show -s $GITHUB_SHA --format='%cI')" >> $GITHUB_ENV
+        env:
+          COMMITTED_AT_TEST: $(git show -s $GITHUB_SHA --format="%cI")
+
```

Comment:
This was added to check the date time but per https://github.com/WordPress/wordpress-develop/blob/trunk/.github/workflows/performance.yml#L36 it use `trunk` file so it will not add these new test step.

Before committing this PR we needs to remove these step.

Reply:
@desrosj @swissspidy Does anyone know how we can check if it's working before committing?

Reply:
Got it and checked in https://github.com/WordPress/wordpress-develop/pull/8072/commits/2b7099cd7270fb1793d06f782329dec388a04371